Source/ZoomNet/Models/ReportParticipant.cs

```diff
@@ -0,0 +1,103 @@
+using Newtonsoft.Json;
+using System;
+
+namespace ZoomNet.Models
+{
+	/// <summary>
+	/// Metrics of a participant.
+	/// </summary>
+	public class ReportParticipant
```

Comment:
I think you can derive from the Participant model class and omit the id, name and email properties in this class.

Reply:
I ended up deriving from Participant and then having ReportMeetingParticipant derive from ReportParticipant as the webinar vs. meeting models were off by one field.